.gitignore

```diff
@@ -3,3 +3,4 @@ build/
 build-cjs/
 coverage/
 node_modules/
+*.tgz
```

Comment:
What is this for?

Reply:
As part of my workflow to test updates in other places, I create a tarball of the build output using `yarn pack` and I install that in other repos. `yarn pack` creates a `enzyme-adapter-preact-pure-4.0.1.tgz` file at the root of the repo so I added *.tgz to the .gitignore to prevent myself from checking it in. I figured it was safe to ignore all `tgz` files since we probably want to generally avoid checking in binary files in this repo.